src/releng_docs/services_shipit.rst

```diff
@@ -63,8 +63,45 @@ It does the following tasks on every run:
 
 
 ``src/shipit_pipeline``
+-----------------------
 
-TODO
+Ship It Pipeline is a service that manages running pipelines. A pipeline is a series of steps that can depend on one
+another and together represent some kind of workflow or process. The pipeline service is responsible for creating
+instances of individual steps on the appropriate step service once that step is runnable. A step is runnable when all
+its dependencies have finished.
+
+A pipeline specifies the backend service URL and inputs for each step in the pipeline. This is so that the pipeline
+service knows how to create specific steps when necessary.
+
+Pipelines and Firefox releases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+NB. This section was written prior to having any pipelines implemented.
+
+Initially we will integrate the legacy ship-it/release-runner infrastructure with the new pipelines.
+
+1. release-runner will create a set of tasks in Taskcluster as normal
+2. release-runner will then create a new pipeline and POST it to the pipeline service for execution. The pipeline will
+   look roughly like this:
+
+    B -> S -> P
```

Comment:
you can use [blockdiag](http://blockdiag.com/en/blockdiag/index.html) if you want (it was recently added to releng_docs.